core-bundle/src/Resources/contao/library/Contao/Model/Registry.php

```diff
@@ -153,12 +165,12 @@ public function register(Model $objModel)
 
 		$strTable = $objModel->getTable();
 
-		if (!\is_array($this->arrAliases[$strTable]))
+		if (!isset($this->arrAliases[$strTable]))
 		{
 			$this->arrAliases[$strTable] = array();
 		}
 
-		if (!\is_array($this->arrRegistry[$strTable]))
+		if (!\is_array($this->arrRegistry[$strTable] ?? null))
```

Comment:
Same here …

Reply:
Sorry, I didn't look at the file after applying the suggestion.